sparkplug/cli.py

```diff
@@ -147,28 +105,7 @@ def main(
     if options.fork:
         executor = sparkplug.executor.Subprocess(options.fork)
 
-    if options.daemon:
```

Comment:
this might warrant a major version increment?

Reply:
I really don't think so.  It's only one feature, and it's not a feature that we have used in years.  I don't think there are many people out in the wild who would be using it either, given the rise of virtualized infrastructure.